import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/helper/DocumentMapper.java

```diff
@@ -155,14 +165,39 @@ private void mapProjects(List<InternalDocument> sourceDocuments, List<InternalDo
 				InternalDocument destDoc = optionalDestDoc.get();
 				String destinationIdValue = (String) destDoc.getStringFields().get(destinationIdField);
 				destDoc.setIdentifier(destinationIdValue);
-				this.addDocumentMapping(srcDoc, destDoc, OperationType.UPDATE);
-			} else {
-				this.addDocumentMapping(srcDoc, null, OperationType.INSERT);
-			}
+				this.addDocumentMapping(srcDoc, destDoc, OperationType.UPDATE, null);
```

Comment:
try to overload the method addDoucmentMapping without the last parameter, by passing the null to the method that has that parameter.